.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+version: 2.1
+
+jobs:
+    test_py3:
+        docker:
+            - image: kkiesling/pymoab-py3-18.04
+        steps:
+            - checkout
+            - run:
+                working_directory: tests/
+                command: |
+                    pip install -e .. --user
```

Comment:
I would separate the test and the pip instal.
maybe just add a:
`- run: pip install -e . --user` after the `- checkout` ?